kong/pdk/ctx.lua

```diff
@@ -52,33 +52,35 @@ local _CTX_CORE_KEY = {}
 
 
 ---
--- A table that has the lifetime of the current request - Unlike
+-- A table that contains the entire lifetime of the current request. Unlike
```

Comment:
Unfortunately we're still deviating from the actual meaning quite a bit with this iteration. What do you think of something like this?

```suggestion
-- A table that has the same lifetime as the current request. Unlike
```

It's not the best sentence in the world, but to the target audience of these docs I believe it conveys its meaning pretty well. I think it's also maybe a tad more readable than the original, but I'd also be fine with reverting to the original line and just adding the period after `request.`

Ditto for the `kogn.ctx.shared` doc block above on line 17.

Reply:
Fixed in https://github.com/Kong/kong/pull/8287/commits/2d57cad679108453d56433bf26b924974ff15f05.